config/constants.js

```diff
@@ -6,6 +6,6 @@ const StageTypes = {
   INACTIVE: "INACTIVE"
 };
 
-const YEAR = "2019";
+const EVENT = "2019";
```

Comment:
In the future I would like this to be removed and rather use the getCurrentEvent util.

Reply:
Ye that would be for the best, but for now getting a working segment out the door is priority in my mind

Reply:
Agreed!